### ISSUE TITLE
Crashloops - exec user process caused: exec format error

### ISSUE THREAD

**Issue by sjoneill** [citrix/citrix-k8s-ingress-controller#445]
Seeing this error when trying to deploy CIC on a ARMv7 CPU architecture in a lab environment.
From my research I've found other ingress controllers experienced similar issues. And it was found the ingress images were incompatible with arm architecture.

`kubectl get pod cic-k8s-ingress-controller-6f99f7d6d5-44ps6
NAME                                          READY   STATUS             RESTARTS   AGE
cic-k8s-ingress-controller-6f99f7d6d5-44ps6   0/1     CrashLoopBackOff   26         114m`

`kubectl logs -f cic-k8s-ingress-controller-6f99f7d6d5-44ps6
standard_init_linux.go:228: exec user process caused: exec format error`

Yaml config file:
`apiVersion: apps/v1
kind: Deployment
metadata:
  name: cic-k8s-ingress-controller
spec:
  selector:
    matchLabels:
      app: cic-k8s-ingress-controller
  replicas: 1
  template:
    metadata:
      name: cic-k8s-ingress-controller
      labels:
        app: cic-k8s-ingress-controller
      annotations:
    spec: 
      serviceAccountName: cic-k8s-role
      containers:
      - name: cic-k8s-ingress-controller
        image: "quay.io/citrix/citrix-k8s-ingress-controller:1.7.6"
        env:
        # Set NetScaler NSIP/SNIP, SNIP in case of HA (mgmt has to be enabled) 
         - name: "NS_IP"
           value: "192.168.X.X"
         # Set username for Nitro
         - name: "NS_USER"
           value: "nsroot"
         # Set user password for Nitro
         - name: "NS_PASSWORD"
           value: "X"
         # Set log level
         - name: "EULA"
           value: "yes"
        args:
          - --ingress-classes
            ingress-class-1
          - --feature-node-watch
            false
        imagePullPolicy: Always
`

**Comment by ankits123** [comment]
Yes that's right .Ingress image is incompatible with arm architecture.

**Comment by mayurmohanpatil** [comment]
Hello @sjoneill we would like to understand your use case for further assistance.
Can we please add your inputs on AppModernization@citrite.net
